### ISSUE TITLE
Cargo.toml members can not use *

### ISSUE THREAD

**Issue by SKTT1Ryze** [rust-lang/cargo#8861]
If I write Cargo.toml like this:  
```
workspace
members = [
    "project_1",
    "project_2",
    "project_3/*",
```
And the `project_3` have some directories and files, then the `cargo build` will be error.  
Sometimes I wish to place readme.md in project_3 directory, at the sometime I want to write `Cargo.toml` with "*", then this problem occurs.  
I think the `cargo` should know the readme.md is a file instead of directory, and the error above shouldn't occur.  
Wish rust to be better.

**Comment by ehuss** [comment]
Which version of Cargo are you using? This should have been fixed in 1.47 via #8511.

**Comment by SKTT1Ryze** [comment]
I just use cargo 1.47.0 (f3c7e066a 2020-08-28).

**Comment by SKTT1Ryze** [comment]
Yes, this has been fixed in 1.47.  
I use cargo 1.45.0-nightly (9fcb8c1d2 2020-05-25).  
Sorry.
